istio/tests/cases/README.md

```diff
@@ -0,0 +1,170 @@
+### Prerequisites
+
+Make sure below requirements are met:
+
+| Version | Name | Details |
+| --- | ------ | ------ |
+|1.9|Kubernetes cluster|Without alpha feature, [link](https://istio.io/docs/setup/kubernetes/quick-start.html#google-kubernetes-engine)|
+|0.6.0|Istio|[link](https://istio.io/docs/setup/kubernetes/quick-start.html)|
+|0.6.0|nginMesh|[link](https://github.com/nginmesh/nginmesh/blob/master/README.md)|
+|1.5.0|Bookinfo Application|[link](https://github.com/istio/istio/blob/master/samples/bookinfo/src)|
+|0.9.2|Mamba|[link](https://github.com/nestorsalceda/mamba)|
+|4.1.0|Wrecker|[link](https://github.com/wg/wrk)|
+|2.7|Python|[link](https://www.python.org)|
+|1.10.0|Kubectl|[link](https://kubernetes.io/docs/tasks/tools/install-kubectl/)|
+
+### Run Test 
+1. Change directory to /cases directory in nginMesh repo:
+```
+cd tests/cases
+```
+2. Install python dependencies in virtual environment:
+```
+./install.sh
+```
```

Comment:
update the README to use virtual environment.

Reply:
ok